framework/src/meshgenerators/FillBetweenCurvesGenerator.C

```diff
@@ -161,6 +161,39 @@ FillBetweenCurvesGenerator::generate()
                                                                  _bias_parameter,
                                                                  _sigma);
 
+  // Transfer over the nodesets
```

Comment:
What about sidesets? Should these operations go in the utility function?

Reply:
no sidesets on 1D curves

Reply:
These operations in this file are pretty specific. I need the PL wherehas I can see other users not need to locate the node (because they have better, not renumbered, access to the previous mesh?)